### DISCUSSION TITLE
Sequence missing

### DISCUSSION THREAD

**Discussion by Shradsruhela03** [rahulnyk/knowledge_graph#33] [General]
Sequence to run this repository is missing kindly help me out with the sequence to run this whole repo

**Comment by rahulnyk** [comment]
Please take a look at these two articles. 

https://medium.com/towards-data-science/how-to-convert-any-text-into-a-graph-of-concepts-110844f22a1a 

https://medium.com/towards-data-science/text-to-knowledge-graph-made-easy-with-graph-maker-f3f890c0dbe8

**Comment by Shradsruhela03** [comment]
I understood the approach but i don't understand the sequence in which i
should run this whole repository.
Please provide the sequence of the files in which i should run in my local
machine
